lib/puppet/provider/jail/iocell.rb

```diff
@@ -0,0 +1,247 @@
+require 'tempfile'
+
+Puppet::Type.type(:jail).provide(:iocell) do
```

Comment:
I assume this work is based on the current iocage provider.  What other changes should I be looking out for?

Reply:
Indeed. Shouldn't be any between iocage legacy and iocell, at least not as of yet. I couldn't find any real differences that would impact.